docs/en/reference/attributes-reference.rst

```diff
@@ -668,11 +668,6 @@ and in the Context of a :ref:`#[ManyToMany] <attrref_manytomany>`. If this attri
 are missing they will be computed considering the field's name and the current
 :doc:`naming strategy <namingstrategy>`.
 
-The ``#[InverseJoinColumn]`` is the same as ``#[JoinColumn]`` and is used in the context
```

Comment:
I wasn’t sure if I should remove only the last sentence or the whole paragraph.

If it’s not necessary, then there’s no need to mention it.